test/e2e/multiple_vols_pod_io/multiple_vols_pod_io_test.go

```diff
@@ -0,0 +1,178 @@
+package multiple_vols_pod_io_test
+
+import (
+	"e2e-basic/common"
+	"e2e-basic/common/e2e_config"
+	"fmt"
+	"strings"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	coreV1 "k8s.io/api/core/v1"
+	storageV1 "k8s.io/api/storage/v1"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var defTimeoutSecs = "120s"
+
+func TestMultipleVolumeIO(t *testing.T) {
+	// Initialise test and set class and file names for reports
+	common.InitTesting(t, "Multiple volumes single pod IO tests", "multiple-volume-pod-io")
+}
+
+func multipleVolumeIOTest(replicas int, volumeCount int, protocol common.ShareProto, volumeType common.VolumeType, binding storageV1.VolumeBindingMode) {
+	logf.Log.Info("MultipleVolumeIOTest", "replicas", replicas, "volumeCount", volumeCount, "protocol", protocol, "volumeType", volumeType, "binding", binding)
+	//	scName := "multi-vol-io-test-" + string(protocol)
```

Comment:
Should this be removed? If it is a comment on the below line it doesn't seem to match the formatting i.e. it is missing the volumeType and binding.

Reply:
Oops, yes removed now.